### PR TITLE
Wup 0000 fixes for various proto files

### PR DESCRIPTION
**Pull request by apavlinovic**
- Added bat file for quick testing 
- Commented out stuff that breaks compile
- Modified some bindings that seem broken

### CODE REVIEW

File: src/WUProtos/Data/GmTemplate.proto [hunk 1]
```diff
@@ -51,14 +51,13 @@ import "WUProtos/Data/Version.proto";
 message GmTemplate {
 	oneof MessageType {
 		.WUProtos.Data.Collection.CollectionFamily collection_family = 1;
-		.WUProtos.Data.Collection.CollectionItem collection_item = 2;
 		.WUProtos.Data.Encounter.Encounter encounter = 3;
 		.WUProtos.Data.Encounter.EncounterObstacle encounter_obstacle = 4;
 		.WUProtos.Data.Spell spell = 5;
 		.WUProtos.Data.VaultItem vault_item = 6;
-		.WUProtos.Data.GameConfig game_config = 7;
+		// .WUProtos.Data.GameConfig game_config = 7;
```
Comment (Furtif) [inline src/WUProtos/Data/GmTemplate.proto:57]: ?!

File: src/WUProtos/Data/Potion/PotionReward.proto [hunk 2]
```diff
@@ -1,10 +1,10 @@
 syntax = "proto3";
 package WUProtos.Data.Potion;
 
-import "WUProtos/Data/Loot/LootCollection.proto";
+// import "WUProtos/Data/Loot/LootCollection.proto";
 
 message PotionReward {
-	.WUProtos.Data.Loot.LootCollection rewards = 1;
+	// .WUProtos.Data.Loot.LootCollection rewards = 1;
```
Comment (Furtif) [inline src/WUProtos/Data/Potion/PotionReward.proto]: !

**Comment by Furtif** [comment]
same issues from https://github.com/Furtif/WUProtos/pull/2

File: src/WUProtos/Data/GmTemplate.proto (hunk 1 above)
Comment (apavlinovic) [reply src/WUProtos/Data/GmTemplate.proto:57]: Doesn't compile with this active

Comment (Furtif) [reply src/WUProtos/Data/GmTemplate.proto:57]: but is false this exists realy into apk

**Pull request closed without merging**